docs/getting_started/overview.rst

```diff
@@ -0,0 +1,53 @@
+************
+Overview
+************
+
+The ``trifinger_simulation`` package provides an interface to the TriFinger robots
+in simulation using the `pybullet physics engine <https://pypi.org/project/pybullet/>`_.
+
+.. note::
+    The TriFinger robots are a little family of kinematically similar robots, including the
+    TriFingerOne, the TriFingerEdu, and the TriFingerPro. These robots signify the different iterations
+    towards making a low-cost, robust, reproducible, and open-source robotic platform, which is both durable
+    against the heavy wear-and-tear of exploration-heavy reinforcement learning algorithms, and also allows
+    for precise control. Among these three, the TriFingerEdu is completely open-source and can be built from
+    scratch. For details, please refer `here <https://github.com/open-dynamic-robot-initiative/open_robot_actuator_hardware/blob/master/mechanics/tri_finger_edu_v1/README.md>`_.
+    You can also find other details on this platform in our `preprint <https://arxiv.org/abs/2008.03596>`_.
+
+    .. list-table:: 
+
+    * - .. figure:: ../images/edu.png
+           :width: 620
+
+           Fig 1. TriFingerEdu
+
+    * - .. figure:: ../images/one.png
+           :width: 620
+
+           Fig 2. TriFingerOne
+
+    * - .. figure:: ../images/pro.png
+           :width: 620
+
+           Fig 3. TriFingerPro
+
+A. Its main components include:
+
+    1. The ``SimFinger`` class: which provides the complete simulation environment for the TriFinger, including all
+    methods to control it, interact with it, to set its properties, as well as to setup the world around it.
+    For more details on this class, check out :doc:`../api/sim_finger` (API doc).
```

Comment:
This list is currently not rendered as an actual list.  I think it is because the following lines are not indented.
```suggestion
    1. The ``SimFinger`` class: which provides the complete simulation environment for the TriFinger, including all
       methods to control it, interact with it, to set its properties, as well as to setup the world around it.
       For more details on this class, check out :doc:`../api/sim_finger` (API doc).
```